File: js/find_digit.js
function find_digits(n) {
  counter = 0;
  number_array = n.toString().split("");
  for (i = 0; i < number_array.length; i++) {
    if (number_array[i] != 0 && n % number_array[i] == 0) {
      counter++;
    }
  }
  return counter;
}

console.log(find_digits(25));

File: ruby/find_digit.rb
def find_digits(n)
  counter = 0
  number_array = n.to_s.split("")
  number_array.each { |x|  counter += 1 if x.to_i != 0 && n % x.to_i == 0}
  puts counter
end


puts find_digits(25)